modules/digiTrustIdSystem.js

```diff
@@ -83,8 +83,7 @@ function writeDigiId(id) {
   var key = 'DigiTrust.v1.identity';
   var date = new Date();
   date.setTime(date.getTime() + 604800000);
-  var exp = 'expires=' + date.toUTCString();
-  document.cookie = key + '=' + encId(id) + '; ' + exp + '; path=/;SameSite=none;';
+  utils.setCookie(key, encId(id), date.toUTCString(), 'none');
```

Comment:
this may require more refactoring because `encId()`, calls encodeURIComponent() and so does `setCookie()`.  encoding twice could be a problem if it's not decoded twice on the other end.   So maybe we refactor `encId()` or maybe add an option to `setCookie()` to NOT encode.

Reply:
Let me look into this. The id is typically passed through and decoded via a server-side process. I'll see if there is a reasonable place to account for possible double encoding in that pipeline.

Reply:
@goosemanjack - would like to get this ticket merged. Should we remove the call to encId()?

Reply:
I made a change here. Removed the call to encodeURI from encId. @goosemanjack - please review when you get a chance, but we need to merge this PR.